Tools/AP_Periph/rc_out.cpp

```diff
@@ -24,20 +24,18 @@
 #define SERVO_OUT_RCIN_MAX      16  // SRV_Channel::k_rcin1 ... SRV_Channel::k_rcin16
 #define SERVO_OUT_MOTOR_MAX     12  // SRV_Channel::k_motor1 ... SRV_Channel::k_motor8, SRV_Channel::k_motor9 ... SRV_Channel::k_motor12
 
-#if HAL_PWM_COUNT == 0
```

Comment:
What is this accomplishing in this particular PR? What is this new board using from the servo library? SRV_Channel.h has this logic which will cause it to have 16 servos and thus a bunch of params that do nothing..?

```
#if !defined(NUM_SERVO_CHANNELS) && defined(HAL_BUILD_AP_PERIPH) && defined(HAL_PWM_COUNT) && (HAL_PWM_COUNT >= 1)
    #define NUM_SERVO_CHANNELS HAL_PWM_COUNT
#else
    #define NUM_SERVO_CHANNELS 16
#endif
```

The reason I put in the HAL_PWM_COUNT==0 check is if NUM_SERVO_CHANNELS=0 then all hell breaks loose throughout all other libraries that use NUM_SERVO_CHANNELS and lots of changes are need to wrap them to allow=0. Did you run into any of that? Maybe that's been handled since then?

Reply:
its needed for notify for LEDs